.eslintrc.js

```diff
@@ -232,6 +232,7 @@ module.exports = {
     '@typescript-eslint/ban-types': 0,
     '@typescript-eslint/restrict-template-expressions': 0,
     '@typescript-eslint/no-shadow': 2,
+    '@typescript-eslint/no-non-null-assertion': 0,
```

Comment:
Finally giving in... This greatly simplifies the usage of props that have default values, by using `someOptionalProp!.length` instead of `(someOptionalProp as string[]).length`